io/src/test/scala/sbt/io/CopyDirectorySpec.scala

```diff
@@ -25,6 +25,6 @@ class CopyDirectorySpec extends FlatSpec with Matchers {
     IO.copyDirectory(dir / "src", dir / "dst")
 
     // Then: dst/lib/a.txt should have been created and have the correct contents
-    IO.read(dir / "dst" / "lib" / "a.txt") shouldBe "this is the file contents"
+    assert(IO.read(dir / "dst" / "lib" / "a.txt") == "this is the file contents")
```

Comment:
What's the motivation for moving away from the scalatest dsl to the more traditional assertion syntax?

Reply:
ScalaTest has always supported `assert(...)` (http://www.scalatest.org/user_guide/using_assertions), but my overall preference has lately been moving away from BDD style to simpler unit tests, preferably with DiagrammedAssertions or [Expecty](http://eed3si9n.com/power-assert-with-expecty). I think they are easier to read/write/maintain.

Reply:
Makes sense. I was just curious if this was a settled style decision moving forward. I don't have a horse in the race.